cli/hclfmt.go

```diff
@@ -93,6 +94,14 @@ func formatTgHCL(terragruntOptions *options.TerragruntOptions, tgHclFile string)
 
 	fileUpdated := !bytes.Equal(newContents, contents)
 
+	if terragruntOptions.Diff && fileUpdated {
+		diff, err := bytesDiff(contents, newContents, tgHclFile)
+		if err != nil {
+			return fmt.Errorf("Failed to generate diff for %s", tgHclFile)
+		}
+		fmt.Fprintf(terragruntOptions.Writer, "%s\n", diff)
```

Comment:
Unhandled returned error

Example:
https://github.com/gruntwork-io/terragrunt/blob/master/cli/cli_app.go#L390

Reply:
Hm, maybe I miss something but error returned from `fmt.Fprintf()` is not handled which may lead to more issues if left unhandled.

https://pkg.go.dev/fmt#Fprintf

Reply:
Sorry, I thought you meant line 100, not 102. Going to fix.